.github/workflows/node.js.yml

```diff
@@ -24,5 +24,5 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - run: npm ci
-    - run: npm run build
+    - run: yarn ci
```

Comment:
Does yarn supports `yarn ci`?

Reply:
No just got to know Github does not support and certified yarn.